tests/fractional_scale_v1.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2020 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "expect_protocol_error.h"
+#include "generated/fractional-scale-v1-client.h"
+#include "in_process_server.h"
+#include "version_specifier.h"
+#include "wl_handle.h"
+
+using namespace wlcs;
+using testing::Eq;
```

Comment:
Don't really need this anymore?

Reply:
Only the include is not needed. the namespace helps a bit with code nose and `testing::Eq` is required for `EXPECT_PROTOCOL_ERROR`.